src/effects/effectchainslot.h

```diff
@@ -35,6 +35,7 @@ class EffectChainSlot : public QObject {
 
     void loadEffectChain(EffectChainPointer pEffectChain);
     EffectChainPointer getEffectChain() const;
+    EffectChainPointer getAndEnsureEffectChain(EffectsManager* pEffectsManager);
```

Comment:
I don't understand what "ensure" means just from the name of the function without knowing the implementation. Not a good function name.